### ISSUE TITLE
Fix: ICD10CM codes previously mapped to 'Patient encounter procedure' and value are mapped directly to a procedure or event

### ISSUE THREAD

**Issue by dimshitc** [OHDSI/Vocabulary-v5.0#996]
The Maps to 'Patient encounter procedure' get deprecated, most of the 'Maps to value' relationships turn into 'Maps to'. Some relationships were improved and added. This makes a direct mapping to the event or procedure instead of postcoordinated expression.
Please use this community contribution file 
https://docs.google.com/spreadsheets/d/1Ft9A3Wafldm1jzLSuwXpow18F711nzKs/edit#gid=60139203

**Comment by Alexdavv** [comment]
@dimshitc are you describing the issue or some idea?

And wait a minute. Didn't we say the encounter for a procedure doesn't automatically mean the procedure but only an intent to perform the procedure?

**Comment by dimshitc** [comment]
No, we agreed already that we need to map it to procedure or event directly.
I'm going to attach the community contribution file here (the link to GD), But I need to create the issue first, because the name of the file requires the number of issue

**Comment by cgreich** [comment]
Agree with @dimshitc. The fact that the procedure was done during an "encounter" is pretty self-evident. Plus we should avoid "heavy" values as concepts (except in history and family history). Nobody ever looks there.

Of course, some of them are useless one way or another: 45537675 "[Health supervision and care of other healthy infant and child](https://athena.ohdsi.org/search-terms/terms/45537675)" would be mapped (and is now mapped to value) 4145320 "[Child health care](https://athena.ohdsi.org/search-terms/terms/4145320)". May as well map it to 4125559 "[Nothing](https://athena.ohdsi.org/search-terms/terms/4125559)".

**Comment by m-khitrun** [comment]
Hi, @dimshitc! We improved the mappings of Patient encounter codes. Thank you for your contribution.
